package.json

```diff
@@ -14,6 +14,8 @@
     "@commitlint/cli": "^9.0.1",
     "@commitlint/config-conventional": "^9.0.1",
     "@commitlint/travis-cli": "^9.0.1",
+    "@swc/core": "^1.2.6",
```

Comment:
Not sure if we need the added dependencies. Or am i missing something?

Reply:
That's probably by mistake.

Reply:
I reverted such changes.